com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDRenderPipelineUI.cs

```diff
@@ -383,6 +383,15 @@ static void Drawer_SectionShadows(SerializedHDRenderPipelineAsset serialized, Ed
                 serialized.renderPipelineSettings.hdShadowInitParams.shadowResolutionPunctual.ValueGUI<int>(Styles.punctualLightsShadowTiers);
                 EditorGUILayout.DelayedIntField(serialized.renderPipelineSettings.hdShadowInitParams.maxPunctualShadowMapResolution, Styles.maxShadowResolution);
                 --EditorGUI.indentLevel;
+
+                ++EditorGUI.indentLevel;
+                // Because we don't know if the asset is old and had the cached shadow map resolution field, if it was set as default float (0) we force a default. 
+                if (serialized.renderPipelineSettings.hdShadowInitParams.cachedPunctualShadowAtlasResolution.intValue == 0)
+                {
+                    serialized.renderPipelineSettings.hdShadowInitParams.cachedPunctualShadowAtlasResolution.intValue = 2048;
```

Comment:
if we do that we will allocate more memory for title, even if they don't use cached shadow.
I am wondering if we should have a default of 0 (so no allocation) and a message on light that say  that atlas is 0 if someone use onEnable, or a warning somewhere in the editor when we try to allocate a cached shadow and it doesn't fit in the atlas?

I am a bit worryed by all the wasted memory we have with all the atlas that we have: )

Reply:
Doing this would break all existing projects using the cached shadows though :( 

I guess a message on the light can be done, but then people need to go on the light to see the issue.  

We cannot put anything in log without it being a spam and I thought we were not allow to spam warnings every frame.